.eslintrc.js

```diff
@@ -47,15 +47,13 @@ module.exports = {
       files: ['test/**/*'],
       rules: {
         'import/no-extraneous-dependencies': 'off',
-        'import/no-named-as-default': 'off',
       },
     },
     {
       files: ['**/__tests__/**'],
       rules: {
         'functional/no-expression-statement': ['off'],
         'import/no-extraneous-dependencies': ['off'],
-        'import/no-named-as-default': 'off',
```

Comment:
@sarahdayan with was the last edit from a previous comment. To fix this ESLint issue rather than ignore it.